framework/Source/Kirakira.swift

```diff
@@ -196,12 +208,14 @@ public class Kirakira: OperationGroup {
 
         self.configureGroup { input, output in
             input
+            --> blendImageRescaleEffect
             --> sparklesEffect
             --> blurEffect
             --> saturationEffect
             saturationEffect.addTarget(addBlend, atTargetIndex: 1)
 
             input
+            --> sourceImageRescaleEffect
```

Comment:
Only apply RescaleEffect for creating sparkle images to sync the results of different image scales.
There is no need to change the input size.